<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgAviRecorder</class>
 <widget class="QDialog" name="CQDlgAviRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Recorder</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="launchAtSimulationStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Launch at next simulation start</string>
   </property>
  </widget>
  <widget class="QToolButton" name="recordNow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Record now!</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showCursor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Show cursor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showButtons">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Show button states</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hideInfos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Hide information text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileLocation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="selectLocation">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="displayedFramesVsRecordedFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>displayed frames result in 1 recorded frame</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame rate [f/s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frameRate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="qqOutputTypeCombo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Output type</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoFrameRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Automatic frame rate</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="recordDesktopInstead">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Record desktop content</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="recordWindowInstead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Record window content</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>launchAtSimulationStart</tabstop>
  <tabstop>recordNow</tabstop>
  <tabstop>recordWindowInstead</tabstop>
  <tabstop>recordDesktopInstead</tabstop>
  <tabstop>showCursor</tabstop>
  <tabstop>showButtons</tabstop>
  <tabstop>hideInfos</tabstop>
  <tabstop>fileLocation</tabstop>
  <tabstop>selectLocation</tabstop>
  <tabstop>displayedFramesVsRecordedFrame</tabstop>
  <tabstop>qqOutputTypeCombo</tabstop>
  <tabstop>autoFrameRate</tabstop>
  <tabstop>frameRate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
